velox/functions/prestosql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -18,10 +18,15 @@
 #include <string>
 #include "velox/common/base/tests/GTestUtils.h"
 #include "velox/external/date/tz.h"
+#include "velox/external/duckdb/duckdb.hpp"
 #include "velox/functions/prestosql/tests/utils/FunctionBaseTest.h"
 #include "velox/functions/prestosql/types/TimestampWithTimeZoneType.h"
 #include "velox/type/tz/TimeZoneMap.h"
 
+#include "velox/external/duckdb/duckdb.hpp"
```

Comment:
The DuckdB file includes shouldn't be needed.  Don't think QueryPlanner.h is needed as well. 

Also, the coding guidelines suggest to keep the includes in alphabetical order.